src/main/lnd2glcMod.F90

```diff
@@ -204,7 +204,7 @@ subroutine update_lnd2glc(this, bounds, num_do_smb_c, filter_do_smb_c, &
       ! Make sure we haven't already assigned the coupling fields for this point
       ! (this could happen, for example, if there were multiple columns in the
       ! istsoil landunit, which we aren't prepared to handle)
-      if (fields_assigned(g,n)) then
+      if (1==2) then
```

Comment:
Looking through the surface roughness mods, I came across this modified if-statement.

I don't know the reason for the change, but @ekluzek I'm wondering whether you would approve this change.

Reply:
This looks like a mistake to me. So I'm glad you noticed and pointed it out. I would put it back and make sure things work. If not, we'll need to investigate further. My hope is that this was a test that was accidentally committed, and shouldn't have been. If it is needed it would need to be changed.

Reply:
This needs to be worked on, we need to make sure the if statement is correct. This might take some time to make sure it's the right statement.

Although I wonder if just reverting to

`if (fields_assigned(g,n) )then`

is what should be done.

Reply:
Yes, seemed most logical to revert this back to what we have in main. I have already done so in this commit:
https://github.com/ESCOMP/CTSM/pull/2045/commits/548e8be5d32dd6dc1d0a9db3ff9a8531c20b7935

Reply:
Oh, awesome. When I brought up the file in github, it must have showed me an earlier version. But, looking at it on Cheyenne I see that this was reverted. Thanks for doing that.